### ISSUE TITLE
A simple deploy on @vercel fails without going nothing

### ISSUE THREAD

**Issue by miguelgargallo** [nuxt-themes/docus#892]
<html><body>

Error: No Output Directory named "dist" found after the Build completed. You can configure the Output Directory in your Project Settings.
--

<span>STATIC_BUILD_NO_OUT_DIR: No Output Directory named "dist" found after the Build completed. You can configure the Output Directory in your Project Settings.</span>
</body>
</html>

**Comment by washirving** [comment]
true. its just don't work. 
if create «dist» folder its deploying but shows 404 page

**Comment by Atinux** [comment]
How did you create the vercel deployment?

**Comment by miguelgargallo** [comment]
Vercel templates, on local labs at home and at works labs

**Comment by miguelgargallo** [comment]
Yeah because mine works perfectly at dnser.pencil.li (personal non-spam example project on docus)

**Comment by Atinux** [comment]
Happy for me to give you access to https://nuxt.studio ?
It's easier to create and edit Docus projects from it :)

**Comment by marcelschilling** [comment]
> They need to update the template since it is actually linking to the wrong one, let me ping them again

What do you mean with "wrong linking" and how could this be fixed?

**Comment by Atinux** [comment]
I am talking with the Vercel team to see how they can update the "deploy" link to link to the correct repository or sub folder

**Comment by Tahul** [comment]
@Atinux is this closed?

I heard in recent meeting the templates are finally fixed?
